builtin/credential/cert/path_certs.go

```diff
@@ -294,6 +326,24 @@ func (b *backend) pathCertWrite(ctx context.Context, req *logical.Request, d *fr
 	if certificateRaw, ok := d.GetOk("certificate"); ok {
 		cert.Certificate = certificateRaw.(string)
 	}
+	if ocspCertificatesRaw, ok := d.GetOk("ocsp_ca_certificates"); ok {
+		cert.OcspCaCertificates = ocspCertificatesRaw.(string)
+	}
+	if ocspEnabledRaw, ok := d.GetOk("ocsp_enabled"); ok {
+		cert.OcspEnabled = ocspEnabledRaw.(bool)
+	}
+	if ocspServerOverrides, ok := d.GetOk("ocsp_servers_override"); ok {
+		cert.OcspServersOverride = ocspServerOverrides.([]string)
+	}
+	if ocspFailOpen, ok := d.GetOk("ocsp_fail_open"); ok {
+		cert.OcspFailOpen = ocspFailOpen.(bool)
+	}
+	if ocspQueryAll, ok := d.GetOk("ocsp_query_all_servers"); ok {
+		cert.OcspQueryAllServers = ocspQueryAll.(bool)
+	}
+	if ocspCacheSize, ok := d.GetOk("ocsp_cache_size"); ok {
+		cert.OcspCacheSize = ocspCacheSize.(int)
+	}
```

Comment:
Also, do we need to re-initialize our OCSP client here with the new values for cache size and enabled/disabled status? :-) 

As it is, I think if you start disabled and then enable it, it might break? And if you change the cache size, I'm not sure that's reflected in the client...

Reply:
(The answer is no, as this was was moved to the global config, not the per-cert role config).